src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageAccountProvider.cs

```diff
@@ -187,6 +188,11 @@ private IStorageAccount StorageAccount
             return account;
         }
 
+        public Task<IStorageAccount> TryGetAccountAsync(string connectionStringName, CancellationToken cancellationToken)
```

Comment:
Recommend removing this new method, and just doing the normal pattern inside the above existing method of checking cache first and returning early if found.